megalinter/descriptors/arm.megalinter-descriptor.yml

```
@@ -45,12 +45,14 @@ linters:
             "
     install:
       dockerfile:
-        - ARG ARM_TTK_NAME='master.zip'
-        - ARG ARM_TTK_URI='https://github.com/Azure/arm-ttk/archive/master.zip'
+        - |
+          # renovate: datasource=github-tags depName=Azure/arm-ttk
+          ARG ARM_TTK_VERSION=20240328
+        - ARG ARM_TTK_NAME='arm-ttk.zip'
```

```suggestion
        - |-
          # renovate: datasource=github-tags depName=Azure/arm-ttk
          ARG ARM_TTK_VERSION=20240328
        - ARG ARM_TTK_NAME='arm-ttk.zip'
```

You can add this at the end to trim the added newline. It is something from the yaml spec.

You don't need to regenerate all, leaving it for the next time

---

> Further enhancement

Done!